docs/en/advanced/airspeed_validation.md

```diff
@@ -0,0 +1,128 @@
+
+If any check fails persistently, the system can declare airspeed as invalid. When this happens, PX4 discards the sensor data and falls back to the option configured by [ASPD_FALLBACK](#aspd_fallback_table).
+
+::: info
+By default, the [Missing Data](#missing-data-check), [Data Stuck](#data-stuck-check), and [Innovation](#innovation-check) checks are enabled.
+You can configure which checks are active using the [ASPD_DO_CHECKS](#aspd_do_checks_table) parameter.
+:::
+
+## Validation Checks
+
+These checks are primarily designed to validate incoming airspeed sensor data and detect potential sensor failures during flight. However, they may also trigger due to configuration issues, estimator inaccuracies, or specific flight conditions that lead to physically inconsistent measurements.
+
+The following overview summarizes each check, common failure causes, and relevant configuration parameters.
+
+### Missing Data Check
+
+::: info
+This check is independent of the other validation checks and cannot be disabled or configured.
+It must pass for any of the other checks to run.
+:::
+
+Triggers when no new airspeed data has been received for more than 1 second.
+
+Common failure causes:
+
+- Faulty or disconnected sensor.
+
+### Data Stuck Check
+
+Triggers when the measured indicated airspeed (IAS) has not changed for more than 2 seconds.
+
+Common failure causes:
+
+- Faulty or blocked airspeed sensor (pitot tube).
+- Very low sensor resolution.
+
+### Innovation Check
+
+Compares the estimated true airspeed (TAS) to the predicted GNSS-based airspeed (groundspeed minus windspeed).
+If the difference exceeds a threshold ([ASPD_FS_INNOV](#aspd_fs_innov_table)) for a prolonged period ([ASPD_FS_INTEG](#aspd_fs_integ_table)), the check fails.
+
+This check helps detect cases where the measured airspeed is inconsistent with the aircraft’s predicted kinematic state, often indicating sensor drift, (partial) blockage, or sub-optimal sensor placement on the vehicle.
+
+Common failure causes:
+
+- Faulty or blocked airspeed sensor (_sensor fault_)
+- Poor sensor placement on vehicle (_configuration issue_)
+- Poor GNSS data (_estimator innacuracy_)
+- Inaccurate wind estimate (_estimator innacuracy_)
+
+Relevant parameters: [ASPD_FS_INNOV](#aspd_fs_innov_table), [ASPD_FS_INTEG](#aspd_fs_integ_table)
+
+### Load Factor Check
+
+Checks whether the measured airspeed is physically consistent with the aircraft's current load factor.
+If the measured airspeed is too low to plausibly generate the required lift, the sensor reading is considered invalid.
+
+This check helps detect cases where the airspeed sensor may be under-reading during flight.
+
+Common failure causes:
+
+- Faulty or blocked airspeed sensor (_sensor fault_)
+- Uncalibrated airspeed sensor (_configuration issue_)
+- Poor sensor placement on vehicle (_configuration issue_)
+- Incorrect stall speed configuration (_configuration issue_)
+- The vehicle is stalling (_flight condition_)
```

Comment:
This is by far the biggest one. Also when doing manual landings the vehicle naturally "stalls" at the end and the check triggers, that's why it's disabled by default (plus because we have no way of distinguishing it from a in-air stall).